week02/792. Number of Matching Subsequences.cpp

```diff
@@ -0,0 +1,31 @@
+//MC:0(1)
```

Comment:
Brother, this is not `O(1)` in memory complexity. :) You are storing all indices in `seen` eventually!